service/src/main/java/gov/nasa/pds/api/registry/model/api_responses/ProductBusinessLogicImpl.java

```diff
@@ -93,11 +108,15 @@ public static Map<String, List<String>> getFilteredProperties(Map<String, Object
 
     if ((included_fields == null) || (included_fields.size() == 0)) {
       String apiProperty;
+      log.debug("Excluded fields are " + excluded_fields);
       for (Map.Entry<String, Object> entry : sourceAsMap.entrySet()) {
         try {
           apiProperty = SearchUtil.openPropertyToJsonProperty(entry.getKey());
-          if ((excluded_fields == null) || !excluded_fields.contains(apiProperty))
+          log.debug("see if property " + apiProperty + "should be added");
+          if ((excluded_fields == null) || !excluded_fields.contains(apiProperty)) {
+            log.debug("confirmed, the property is being added");
```

Comment:
Are these lines intended to remain in the codebase, or were they temporary dev logs?

Reply:
Right temporary logs. I can remove them.